### PR TITLE
test(integration): include control plane metrics in integration tests

### PR DESCRIPTION
**Pull request by swiatekm-sumo**
##### Description

With some configuration changes to the kind cluster, it's possible to see control plane metrics in integration tests. I've based the tests on ServiceMonitor definitions in values.yaml, and it turns out there's a few metrics which have been deprecated since the original config was written. I've clearly marked them in test definitions and added TODOs to remove them.

---

##### Checklist

Remove items which don't apply to your PR.
- [X] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: tests/integration/yamls/cluster.yaml [hunk 1]
```diff
@@ -2,3 +2,18 @@ kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
 - role: control-plane
+  kubeadmConfigPatches:
```
Comment (pmalek-sumo) [inline tests/integration/yamls/cluster.yaml:5]: Any links to `kind` (or not) docs that treat about this?

Comment (swiatekm-sumo) [reply tests/integration/yamls/cluster.yaml:5]: Kind docs on customizing kubeadm: https://kind.sigs.k8s.io/docs/user/configuration/#kubeadm-config-patches
And kubeadm docs on configuring control plane components: https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/control-plane-flags/

You reckon I should put those in the comment?

Comment (pmalek-sumo) [reply tests/integration/yamls/cluster.yaml:5]: yes, please

**Comment by swiatekm-sumo** [comment]
I'll wait for https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/2041 to be merged to avoid copying the values.

**Review by pmalek-sumo — APPROVED** [review]

**Comment by pmalek-sumo** [comment]
Linter complains about long lines :(

**Pull request merged**